.swiftpm/xcode/xcshareddata/xcschemes/ObjectMapper.xcscheme

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should it be in the repo at all?

Reply:
I don't think it needs to be committed; I'm seeing mixed practise with it but see no reason to maintain these schemes as they get regenerated 🤔